lib/src/test/java/com/auth0/jwt/impl/JsonNodeClaimTest.java

```diff
@@ -328,7 +328,7 @@ public void shouldGetAsMapValue() throws Exception {
 
         assertThat(claim, is(notNullValue()));
         Map map = claim.as(Map.class);
-        assertThat(((Map<String, Object>) map.get("key")), hasEntry("name", (Object) "john"));
+        assertThat(((Map<String, Object>) map.get("key")), hasEntry(Claims.NAME, (Object) "john"));
```

Comment:
Don't use Claims constants

Reply:
i dont see why its a prob for tests

Reply:
Simple put:
```java
//constants (values source)
class Claims {
 NAME="name"
 //...
}

//library code
class Foo {
  void Bar {
     //...
     message.addClaim(Claims.NAME, "John Doe");
  }
}

//test code
class FooTest {
  //...
  assert(message.getClaim(Claims.NAME), is("John Doe"));
}
```

Now go and change Claims.NAME to "namee". 

Will the test pass?

Reply:
there are many more failing tests after its changed to "namee", which is expected, right?